homeassistant/components/luftdaten/config_flow.py

```diff
@@ -0,0 +1,80 @@
+"""Config flow to configure the Luftdaten component."""
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_SCAN_INTERVAL, CONF_SHOW_ON_MAP
+from homeassistant.core import callback
+from homeassistant.helpers import aiohttp_client
+
+from .const import CONF_SENSOR_ID, DEFAULT_SCAN_INTERVAL, DOMAIN
+
+
+@callback
+def configured_sensors(hass):
+    """Return a set of configured Luftdaten sensors."""
+    return set(
+        '{0}'.format(entry.data[CONF_SENSOR_ID])
+        for entry in hass.config_entries.async_entries(DOMAIN))
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class LuftDatenFlowHandler(config_entries.ConfigFlow):
+    """Handle a Luftdaten config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    def __init__(self):
```

Comment:
Don't think this is needed, if we don't need it.

Reply:
No, it's not. New flow don't need it.